src/components/WebexRemoteMedia/WebexRemoteMedia.stories.js

```diff
@@ -41,3 +41,11 @@ stories.add('video and audio', () => (
     </WebexDataProvider>
   </div>
 ));
+
+stories.add('warning alert', () => (
```

Comment:
```suggestion
stories.add('error', () => (
```

Reply:
I actually did like this in the first time and then changed later lol

Reply:
Updated